ert_storage/_alembic/alembic/versions/508be3da9c9a_add_observation.py

```diff
@@ -0,0 +1,56 @@
+"""Add observation
+
+Revision ID: 508be3da9c9a
+Revises: a88924af33b0
+Create Date: 2021-03-30 09:06:02.275517
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "508be3da9c9a"
+down_revision = "a88924af33b0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "observation",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("name", sa.String(), nullable=False),
+        sa.Column("x_axis", sa.PickleType(), nullable=False),
```

Comment:
Just wandering why `x_axis` is not `sa.ARRAY(sa.String())`. That's at least how I encode it as mypy

Reply:
You are probably right